tests/test_backend.py

```diff
@@ -213,15 +226,124 @@ def expect_fail(test_case_path):  # type: (str) -> None
+expect_fail('OnnxBackendNodeModelTest.test_resize_upsample_scales_cubic_cpu')
+expect_fail('OnnxBackendNodeModelTest.test_resize_upsample_scales_linear_align_corners_cpu')
+expect_fail('OnnxBackendNodeModelTest.test_resize_upsample_scales_linear_cpu')
+expect_fail('OnnxBackendNodeModelTest.test_resize_upsample_scales_nearest_cpu')
+expect_fail('OnnxBackendNodeModelTest.test_resize_upsample_sizes_cubic_cpu')
+expect_fail('OnnxBackendNodeModelTest.test_resize_upsample_sizes_nearest_ceil_half_pixel_cpu')
+expect_fail('OnnxBackendNodeModelTest.test_resize_upsample_sizes_nearest_cpu')
+expect_fail('OnnxBackendNodeModelTest.test_resize_upsample_sizes_nearest_floor_align_corners_cpu')
+expect_fail('OnnxBackendNodeModelTest.test_resize_upsample_sizes_nearest_round_prefer_ceil_asymmetric_cpu')
+
+# Pad errors - NGONNX-783
+expect_fail('OnnxBackendNodeModelTest.test_constant_pad_cpu')
+expect_fail('OnnxBackendNodeModelTest.test_edge_pad_cpu')
+expect_fail('OnnxBackendNodeModelTest.test_reflect_pad_cpu')
+
+# DepthToSpace CRD mode - NGONNX-784
+expect_fail('OnnxBackendNodeModelTest.test_depthtospace_crd_mode_cpu')
+expect_fail('OnnxBackendNodeModelTest.test_depthtospace_crd_mode_example_cpu')
+
+# DynamicQuantizeLinear - NGONNX-786
+expect_fail('OnnxBackendNodeModelTest.test_dynamicquantizelinear_cpu')
+expect_fail('OnnxBackendNodeModelTest.test_dynamicquantizelinear_expanded_cpu')
+expect_fail('OnnxBackendNodeModelTest.test_dynamicquantizelinear_max_adjusted_cpu')
+expect_fail('OnnxBackendNodeModelTest.test_dynamicquantizelinear_max_adjusted_expanded_cpu')
+expect_fail('OnnxBackendNodeModelTest.test_dynamicquantizelinear_min_adjusted_cpu')
+expect_fail('OnnxBackendNodeModelTest.test_dynamicquantizelinear_min_adjusted_expanded_cpu')
+
+# Range op - NGONNX-787
+expect_fail('OnnxBackendNodeModelTest.test_range_float_type_positive_delta_cpu')
+expect_fail('OnnxBackendNodeModelTest.test_range_float_type_positive_delta_expanded_cpu')
+expect_fail('OnnxBackendNodeModelTest.test_range_int32_type_negative_delta_cpu')
+expect_fail('OnnxBackendNodeModelTest.test_range_int32_type_negative_delta_expanded_cpu')
+
+# Unique op - NGONNX-761
+expect_fail('OnnxBackendNodeModelTest.test_unique_not_sorted_without_axis_cpu')
+expect_fail('OnnxBackendNodeModelTest.test_unique_sorted_with_axis_3d_cpu')
+expect_fail('OnnxBackendNodeModelTest.test_unique_sorted_with_axis_cpu')
+expect_fail('OnnxBackendNodeModelTest.test_unique_sorted_with_negative_axis_cpu')
+expect_fail('OnnxBackendNodeModelTest.test_unique_sorted_without_axis_cpu')
+
+# Gemm-11 - NGONNX-788
+expect_fail('OnnxBackendNodeModelTest.test_gemm_default_no_bias_cpu')
+
+# Round - NGONNX-760
+expect_fail('OnnxBackendNodeModelTest.test_round_cpu')
+
+# Operations not supported by nGraph Backends
+expect_fail('OnnxBackendNodeModelTest.test_top_k_cpu')
+expect_fail('OnnxBackendNodeModelTest.test_top_k_negative_axis_cpu')
+expect_fail('OnnxBackendNodeModelTest.test_top_k_smallest_cpu')
```

Comment:
We should `expect_fail` of these tests only for unsupported backends.

Reply:
Yes, but currently that means all backends, so the code would be repeated 3 times.